playbooks/acs.yml

```diff
@@ -722,24 +722,25 @@
       ansible.builtin.fail:
         msg: "Audit Storage requires ACS 23.4 or later"
       when: not acs_version_requirement
-  tasks:
-    - name: Audit Storage Role import
+  roles:
+    - role: "../roles/java"
       when: acs_is_enterprise and acs_version_requirement
-      ansible.builtin.import_role:
-        name: "../roles/audit_storage"
-      vars:
-        java_version: "{{ acs_play_java_version }}" # noqa: var-naming[no-role-prefix]
-        audit_storage_version: "{{ acs_play_audit_storage_version }}"
-        audit_storage_zip_url: "{{ acs_play_audit_storage_archive_url }}"
-        audit_storage_zip_checksum: "{{ acs_play_audit_storage_archive_checksum }}"
-        audit_storage_username: "{{ username }}"
-        audit_storage_group_name: "{{ group_name }}"
-        audit_storage_broker_url: "failover:({{ activemq_transport }}://{{ activemq_host }}:{{ ports_cfg.activemq[activemq_protocol] }})"
-        audit_storage_broker_username: "{{ activemq_username }}"
-        audit_storage_broker_password: "{{ activemq_password }}"
-        audit_storage_opensearch_url: "{{ elasticsearch_protocol }}://{{ elasticsearch_host }}:{{ ports_cfg.elasticsearch.http }}"
-        audit_storage_opensearch_username: "{{ elasticsearch_username }}"
-        audit_storage_opensearch_password: "{{ elasticsearch_password }}"
+    - role: alfresco.platform.audit_storage
+      when: acs_is_enterprise and acs_version_requirement
+      audit_storage_java_bin_path: "/opt/openjdk-{{ acs_play_java_core }}/bin/java"
```

Comment:
better pass `audit_storage_java_home_path` as you don't really need to change the `bin/java` part - lazy evaluation should do the magic